src/main/java/frc/robot/commands/SwitchSides.java

```diff
@@ -0,0 +1,54 @@
+package frc.robot.commands;
+
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.wpilibj2.command.*;
+import frc.robot.services.ArmGravityCompensation;
+import frc.robot.subsystems.*;
+
+import java.util.function.Supplier;
+
+/**
+ * Switches the side of the arm.
```

Comment:
You can add a funny comment describing what happens to a person if he stands next to the robot while executing this command